src/App.js

```diff
@@ -4,7 +4,18 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    moviesList: [...moviesFromServer],
+  };
```

Comment:
1. On every change save the input value into the `state.query`
1. Filter the movies using `state.query` by searching it in the `movie.title` or `movie.description`